src/machine.rs

```diff
@@ -24,6 +20,13 @@ pub const STACK_ADDR: u64 = 32 * PAGE_SIZE; // not really about the "stack", but
 pub const STACK_SIZE: u64 = 16 * PAGE_SIZE; // whatever
 pub const NUM_CPUS: u64 = 1;
 
+pub struct FrameData<'tcx> {
+    pub call_id: stacked_borrows::CallId,
+    pub catch_panic: Option<UnwindData<'tcx>>,
+    pub is_panic_start: bool
```

Comment:
These fields should all have doc comments.

For `catch_panic`, this is now set on the frame of the function called by `__rust_maybe_catch_panic`, right? And `is_panic_start` is set on the trampoline frame created by `miri_start_panic`?